.github/workflows/pr.yaml

```
@@ -2,57 +1,34 @@
-      pull-requests: write   # Needed for posting a comment
     steps:
       - uses: actions/checkout@v4
         with:
           lfs: true
-      - uses: Gr1N/setup-poetry@v9
-      - uses: actions/setup-python@v5
-        with:
-          python-version: '3.12'
-          cache: poetry
       - name: Get source
         uses: ./.github/actions/get-src
```

actionlint (1.7.7)

description is required in metadata of "GenVM get source" action at "/home/jailuser/git/.github/actions/get-src/action.yaml"

(action)